generators/entity-client/templates/vue/src/test/javascript/e2e/entities/entity-page-object.ts.ejs

```diff
@@ -21,44 +21,54 @@ import { element, by, ElementFinder } from 'protractor';
 import { waitUntilCount, waitUntilDisplayed } from '<% if (clientRootFolder && !skipUiGrouping) { %>../<% } %>../../util/utils';
 
 export default class <%= entityClass %>ComponentsPage {
-    createButton: ElementFinder = element(by.id('jh-create-entity'));
-    deleteButtons = element.all(by.css('div table .btn-danger'));
-    title: ElementFinder = element(by.id('<%= entityFileName %>-heading'));
+  createButton: ElementFinder = element(by.id('jh-create-entity'));
+  detailButtons = element.all(by.css('.btn-info.details'));
```

Comment:
not consistent here, sometimes we use plurial, sometimes not: maybe just a missing 's' for edit button

Reply:
That's because of the selector, sometimes we focus on detailButtons that are listed, and createButton is only here in the header as a single button. It seems logical to me, still do you want me to change it ?

Reply:
maybe just the editButton to editButtons, which is a list, isn't it ?

Reply:
Oh, yes you are right !